core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -436,9 +436,20 @@ oppia.directive('conversationSkin', [
             }
 
             if (ExplorationPlayerStateService.isStateTerminal(stateName)) {
+              $scope.parentExplorationIds =
+                UrlService.getQueryFieldValuesAsList('parent');
+              var recommendedExploration = [];
```

Comment:
This is misnamed, it should be recommendedExplorationIds.

Reply:
Done